wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -152,6 +153,7 @@ public void onCreate() {
         log.info("WalletApplication.onCreate()");
         config = new Configuration(PreferenceManager.getDefaultSharedPreferences(this), getResources());
         autoLogout = new AutoLogout(config);
+        myPackageReplaced = false;
```

Comment:
If the var is already initialized at `false` on its declaration, why it needs to be set as `false` here as well?

Reply:
Removed